### DISCUSSION TITLE
Welcome to common-domain-model Discussions!

### DISCUSSION THREAD

**Discussion by mcleo-d** [finos/common-domain-model#2059] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by vishalpnaikjpm** [comment]
Hello.
I was wondering if some of the Create functions are removed from latest versions of CDM
like Create_Instruction and Create_Counterparty. And if removed what is teh equivalent of that

**Comment by eteridvalishvili** [comment]
Hi @vishalpnaikjpm I just wanted to check in and see if you were able to find the answer you were looking for in the previous message. If not, please let me know and I can help you create an issue on the CDM repository. I can also connect you with the maintainers if necessary. Looking forward to hearing back from you. Thanks!
